fastchat/model/model_exllama.py

```diff
@@ -0,0 +1,76 @@
+import torch
+import gc
+
+from typing import Dict
+
+try:
+    from exllamav2.generator import ExLlamaV2StreamingGenerator, ExLlamaV2Sampler
+except:
+    pass
```

Comment:
1. Move these import statements under `generate_stream_exllama`
2. Raise error if the import is not found